index.d.ts

```diff
@@ -0,0 +1,9 @@
+import {RestClient} from "./types/rest-client";
+import {WebsocketClient} from "./types/websocket-client";
+import DefaultLogger = require("./types/logger");
```

Comment:
Following my comment on `types/logger.d.ts`, this should be `import DefaultLogger = require("./types/logger");`